src/com/spikes2212/gui/DashBoardController.java

```diff
@@ -0,0 +1,69 @@
+package com.spikes2212.gui;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
+import edu.wpi.first.wpilibj.Sendable;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class DashBoardController {
+	private Map<String, Supplier<String>> stringFields;
+	private Map<String, Supplier<Double>> doubleFields;
+	private Map<String, Supplier<Boolean>> booleanFields;
+	private Map<String, Supplier<Sendable>> sendableFields;
```

Comment:
I don't think you need a supplier for Sendable, because Sendables should update on their own (same way commands update on their own when you add them to the dashboard)